service/src/test/java/uk/gov/hmcts/reform/fpl/service/casesubmission/SampleCaseSubmissionTestDataHelper.java

```diff
@@ -0,0 +1,296 @@
+package uk.gov.hmcts.reform.fpl.service.casesubmission;
```

Comment:
data helper classes should not be in main code. Please remove. I am really not a fan

Reply:
it's in the test package. is that what you referring to?

Reply:
ignore 😁